app/src/main/java/org/mozilla/fenix/nimbus/NimbusExperimentsFragment.kt

```diff
@@ -5,81 +5,119 @@
 package org.mozilla.fenix.nimbus
 
 import android.os.Bundle
+import android.view.LayoutInflater
 import android.view.View
-import android.widget.TextView
-import androidx.core.view.isVisible
+import android.view.ViewGroup
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
```

Comment:
We don't use wildcard imports. See https://github.com/mozilla-mobile/firefox-android/tree/main/android-components#style for configuring your Android Studio